.env.example

```diff
@@ -12,6 +12,9 @@ APP_DEBUG=false
 # Use `php artisan key:generate` to generate a random key.
 APP_KEY=ChangeMeBy32KeyLengthOrGenerated
 
```

Comment:
We need to put those a comment above these two new parameters to explain what it does.

I suggest
```
# Prevent information leakage by referring to contacts with hashIds instead of
# the actual IDs used in the database.
```

Reply:
Done